src/operations/search_indexes/create.ts

```diff
@@ -0,0 +1,48 @@
+import type { Document } from 'bson';
+
+import type { Collection } from '../../collection';
+import type { Server } from '../../sdam/server';
+import type { ClientSession } from '../../sessions';
+import type { Callback } from '../../utils';
+import { AbstractOperation } from '../operation';
+
+/** @public */
+export interface SearchIndexDescription {
+  /** The name of the index. */
+  name?: string;
+
+  /** The index definition. */
+  description: Document;
+}
+
+/** @internal */
+export class CreateSearchIndexesOperation extends AbstractOperation<string[]> {
+  constructor(
+    private readonly collection: Collection<any>,
```

Comment:
There isn't a standard pattern we follow for our operations.  For example, the `Find*` operations take a collection and no namespace.  So do the bulk operations.

The reason we haven't had this issue in the past is because we rely on the default `Document` for the collection generic in collections

Reply:
Resolved in another thread, no need for `Collection<any>`